gems/pending/appliance_console/database_configuration.rb

```diff
@@ -221,7 +221,14 @@ def validate!
     end
 
     def start_evm
-      pid = fork { LinuxAdmin::Service.new("evmserverd").enable.start }
+      pid = fork do
+        begin
+          LinuxAdmin::Service.new("evmserverd").enable.start
+        rescue RuntimeError => e
```

Comment:
Should you just capture any StandardError? (of whic RuntimeError is a subclass?)